test/hga/HGA_Regression.ipynb

```diff
@@ -657,40 +655,38 @@
-    "\n",
-    "    expected_avnir_results_ten = {'authority': None,\n",
-    "                                  'cs': None,\n",
-    "                                  'file_type': 'png',\n",
-    "                                  'gcs': None,\n",
-    "                                  'gcs_epsg': None,\n",
-    "                                  'n_bands': 4,\n",
-    "                                  'proj_cs': None,\n",
-    "                                  'proj_epsg': None,\n",
-    "                                  'spatial_extent': None,\n",
-    "                                  'variables': avnir_variables_ten,\n",
-    "                                  'width': 3410,\n",
-    "                                  'height': 3238}\n",
-    "    \n",
-    "    check_request_output(avnir_output_ten[0], expected_avnir_results_ten)\n",
-    "    clean_test_artefacts(avnir_directory_ten)\n",
-    "else:\n",
-    "    print('Skipping test: HGA regression tests not configured for this environment.')\n",
-    "\"\"\""
+    "# if environment_information is not None:\n",
+    "#     granule_id = get_test_granule_id(environment_information['env'], 'G1813212660-ASF',\n",
+    "#                                      'G1244144597-EEDTEST')\n",
+    "#\n",
+    "#     avnir_directory_ten = 'avnir_test_ten'\n",
+    "#    \n",
+    "#     avnir_request_ten = Request(collection=environment_information['avnir_collection'],\n",
+    "#                                 spatial=BBox(-51.0, -0.05, -50.75, 0.25), variables=['Band2'],\n",
+    "#                                 granule_id=granule_id, format='image/png')\n",
+    "#\n",
+    "#     avnir_output_ten = make_request_and_download_result(harmony_client, avnir_request_ten,\n",
+    "#                                                         avnir_directory_ten)\n",
+    "#\n",
+    "#     avnir_variables_ten = {'Band1', 'Band2', 'Band3', 'Band4'}\n",
+    "#\n",
+    "#     expected_avnir_results_ten = {'authority': None,\n",
+    "#                                   'cs': None,\n",
+    "#                                   'file_type': 'png',\n",
+    "#                                   'gcs': None,\n",
+    "#                                   'gcs_epsg': None,\n",
+    "#                                   'n_bands': 4,\n",
+    "#                                   'proj_cs': None,\n",
+    "#                                   'proj_epsg': None,\n",
+    "#                                   'spatial_extent': None,\n",
+    "#                                   'variables': avnir_variables_ten,\n",
+    "#                                   'width': 3410,\n",
+    "#                                   'height': 3238}\n",
+    "#\n",
+    "#     check_request_output(avnir_output_ten[0], expected_avnir_results_ten)\n",
+    "#     clean_test_artefacts(avnir_directory_ten)\n",
+    "# else:\n",
+    "#     print('Skipping test: HGA regression tests not configured for this environment.')"
```

Comment:
Uh, was that supposed to be disabled from before?  Why did it pass? Or wasn't it run?

Reply:
These tests were disabled but used the lazy thing you (quite rightly) don't like of making the content a multiline string, instead of an actual comment. (See L660 and L693 in the old version of the diff)

Reply:
reading diffs fails me again.